docs/links/slack.py

```diff
@@ -0,0 +1,7 @@
+from . import link
+
+link_name = "slack" 
+link_text = "slack" 
```

Comment:
```suggestion
link_text = "Slack" 
```

Names should be capitalised.